src/it/projects/analyze-report/pom.xml

```diff
@@ -59,13 +59,40 @@
       <plugins>
         <plugin>
           <artifactId>maven-dependency-plugin</artifactId>
-          <configuration>
-           <outputXML>true</outputXML>
-           <scriptableOutput>true</scriptableOutput>
-           <verbose>true</verbose>
-          </configuration>
+          <version>@project.version@</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-site-plugin</artifactId>
+          <version>3.10.0</version>
```

Comment:
3.11.0 is in the works which upgrades the entire stack. I recommend to wait for it.

Reply:
IMHO not a good idea. It's not worth delaying this, and it's no harder to update to 3.11.0 before or after this PR.

Reply:
Three days more or less, who cares?

Reply:
This is IT test ... project also override m-site-plugin from parent
 
In such situation will be good to have some possibility to inherit plugin version from parent pom, in other case IT test will have outdated dependency in some time and will be not maintain

So I prefer to update m-site-p in separate PR